### DISCUSSION TITLE
Disordered layout of the png

### DISCUSSION THREAD

**Discussion by zhucan** [louwrentius/fio-plot#125] [Q&A]
![image](https://github.com/louwrentius/fio-plot/assets/13997135/f75391ad-0589-430e-8fed-f9a8a1f606f4)

**Comment by zhucan** [comment]
![image](https://github.com/louwrentius/fio-plot/assets/13997135/e17f497e-97a7-47a9-933c-1661c06e1110)

**Reply by zhucan** [reply]
@louwrentius

**Comment by louwrentius** [comment]
If you can share the data so I can reproduce this, I may be able to fix the table column spacing. 

The code for determining the spacing is a compromise for different scenarios.

So I can't guarantee a fix. 

I remember that I never fixed your intial issue. I'll see what I can do as the proper solution may be complex.

**Reply by zhucan** [reply]
@louwrentius sorry about reply later，only comapre more than 20  devices that can reproduce it.

**Reply by louwrentius** [reply]
To be frank, if I tweak some parameters for the table to fix this, some other layout breaks. 

Comparing more than 8-10 devices at once is the current unenforced limit

**Reply by zhucan** [reply]
Do you think it’s feasible if we adjust the image proportions based on the number of disks being compared?

**Reply by louwrentius** [reply]
Hi, the problem is that the table is generated dynamically and controlling the layout is more dificult than it seems. I was using a formula to calculate the with of the columns based on the text length (# chars) but your example shows that this doesn't always pan out right. The formula is a compromise and works mostly OK for graphs up to maybe 8-10 items to compare. This is all in the table_supporting.py lib. 

If you make a change some test graphs look better, but others get worse.
